### ISSUE TITLE
Stay in last selected month

### ISSUE THREAD

**Issue by bodagrave** [eddiewebb/flarum-calendar#12]
First: That is a really fine piece of software eddieweb, thanks for that!
Second: really difficult to get a speaking header for that issue.

The behaviour at the moment is this: Wenn I go somewhere to next year and create a new event, it switches back to the actual month instead of staying in the month I crated the event. So I need to click back. This could grow annoying if you want to create all event in the next year.
might it be possible to stay in the last selected month?

**Comment by eddiewebb** [comment]
Makes sense. I think that's caused by full calendar behavior, which should be adjustable via the initial date config.

https://fullcalendar.io/docs/initialDate

Need to just provide a value of created event
